src/Parser/DisciplinePriest/Modules/Items/NeroBandOfPromises.js

```diff
@@ -17,19 +17,19 @@ class NeroBandOfPromises extends Module {
   on_byPlayer_heal(event) {
     const spellId = event.ability.guid;
 
-    if (spellId === SPELLS.ATONEMENT1.id || spellId === SPELLS.ATONEMENT2.id) {
+    if (spellId === SPELLS.ATONEMENT_HEAL_NON_CRIT.id || spellId === SPELLS.ATONEMENT_HEAL_CRIT.id) {
       const combatant = this.owner.combatants.players[event.targetID];
       if (!combatant) {
         // If combatant doesn't exist it's probably a pet.
         debug && console.log('Skipping Atonement heal event since combatant couldn\'t be found:', event);
         return;
       }
-      if (!combatant.hasBuff(SPELLS.POWER_WORD_BARRIER_BUFF.id, event.timestamp)) {
-        // N'ero is only active when people are in the bubble
+      if (combatant.hasBuff(SPELLS.ATONEMENT_BUFF.id, event.timestamp)) {
```

Comment:
It's not required, but also impossible as PW:B  is applied by the environment, not the player.

It is good enough at the moment to check for players receiving atonement healing without the atonement buff.